### PR TITLE
Created samples folder, updated artifactId and groupId

### PR DESCRIPTION
**Pull request by anaili**
- Updated POM files.
- Added samples folder.
- demoApp moved to samples/VirtualClassroomSample.
- Added getting started helloworld project (got from [JavaOne example project](https://github.com/bessemHmidi/javaOne2015)

### CODE REVIEW

**Comment by bessemHmidi** [comment]
@anaili , to change the GID you must own the domain, it's a precondition for the maven central repos, the nexus serveur has a very strict check list in front of depencency or archertype deployement. they only aceept a GID that eq to a domain that we own or the github repos, this is why the GID include my name :smile: , but i recently created an orguanisation under github "angularBeans" but we need to create a new repos there , or bying a domain name.

File: angular-beans/pom.xml [hunk 1]
```diff
@@ -5,22 +5,34 @@
...
+			<name>Bessem Hmidi</name>
+			<email>bessem.hmidi.perso@gmail.com</email>
+		</developer>
+	</developers>
 
 	<properties>
 		<maven.compiler.source>1.7</maven.compiler.source>
 		<maven.compiler.target>1.7</maven.compiler.target>
-
-		<!-- <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding> -->
```
Comment (yesbutno) [inline angular-beans/pom.xml:13]: Why did you remove the source encoding? This assumes the platform encoding to be UTF-8 which it might not be. Windows for instance still uses CP-1252.

Comment (anaili) [inline angular-beans/pom.xml:13]: True, I removed this line during code cleanup because we've not been using utf-8 anyway, what's the point of putting a commented tag ? I know It would be better to use UTF-8 to avoid platform-dependency issues but no one claimed (until now haha)

Comment (yesbutno) [inline angular-beans/pom.xml:13]: It wasn't a problem for me either. But part of the time I'm working on Windows and here Maven is complaining about actually using cp-1252 and therefore the build being platform dependent. So you put it back in, just for me? Pretty please?  ;-)
